pages/common/gnmic.md

```diff
@@ -6,20 +6,20 @@
 
 - Request device capabilities:
 
-`gnmic --address {{ip:port}} --insecure capabilities`
+`gnmic --address {{ip:port}} capabilities`
```

Comment:
IMO, it's better to provide a specific ip and port examples because `--address` is a long option and describes the meaning of the following placeholder.

Reply:
I think 127.0.0.1 is the only address that wouldn't work and addresses should be formatted as `address_of_server:port_to_connect`